docs/cli/start/install.md

```diff
@@ -35,13 +37,10 @@ curl -sL https://aws-amplify.github.io/amplify-cli/install-win -o install.cmd &&
 
 </amplify-block-switcher>
 
-> Because we're installing the Amplify CLI globally, you might need to run the command above with `sudo`.
-
-
 ### Pre-requisites for installation
 
-* [Install Node.js®](https://nodejs.org/en/download/) and [NPM](https://www.npmjs.com/get-npm) if they are not already on your machine.
-* Verify that you are running at least Node.js version 10.x and npm version 6.x or greater by running `node -v` and `npm -v` in a terminal/console window
+* If using the NPM installation option, you'll need to [install Node.js®](https://nodejs.org/en/download/) and [NPM](https://www.npmjs.com/get-npm) if they are not already on your machine.
```

Comment:
What's the other option?

Reply:
I think it's referring to the cURL option above (that's hidden in the diff):

```shell
curl -sL https://aws-amplify.github.io/amplify-cli/install | bash && $SHELL
```